transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -297,6 +300,18 @@ public ChannelConfig setAllocator(ByteBufAllocator allocator) {
         return this;
     }
 
+    @Override
+    public BufferAllocator getBufferAllocator() {
```

Comment:
You also want to add a `ChannelOption` for it and update the relevant `getOption` and `setOption` method